debian/changelog

```diff
@@ -1,3 +1,13 @@
+pg-auto-failover (1.0.1) testing; urgency=low
```

Comment:
The `debian/changelog` should be about the debian packaging only, so all we need here is a line with

  * New upstream release.